rust_src/src/multibyte.rs

```diff
@@ -37,16 +37,15 @@ use std::slice;
 use libc::{ptrdiff_t, c_char, c_uchar, c_uint, c_int};
 
 use lisp::ExternalPtr;
-use remacs_sys::{CHAR_MODIFIER_MASK, CHAR_SHIFT, CHAR_CTL, emacs_abort, CHARACTERBITS, EmacsInt,
-                 Lisp_String, error};
+use remacs_sys::{emacs_abort, char_bits, EmacsInt, Lisp_String, error};
 
 pub type LispStringRef = ExternalPtr<Lisp_String>;
 
 // cannot use `char`, it takes values out of its range
 pub type Codepoint = u32;
 
 /// Maximum character code
-pub const MAX_CHAR: Codepoint = (1 << CHARACTERBITS) - 1;
+pub const MAX_CHAR: Codepoint = (1 << (char_bits::CHARACTERBITS as u32)) - 1;
```

Comment:
looks like we have to alias these `char_bits` somewhere, readability quickly degrades here.